taskcluster/test/test_transforms_integration_test.py

```diff
@@ -13,7 +13,7 @@
 from fxci_config_taskgraph.util.integration import (
     FIREFOXCI_ROOT_URL,
     STAGING_ROOT_URL,
-    find_tasks,
+    _find_tasks,
```

Comment:
Given this is imported in multiple places, I wonder if it's worth either removing the _ prefix (and calling the function something else), or adding a `fxci_config_taskgraph.util.integration.clear_cache` function for these use cases.

Reply:
I'm not sure I see any point in wrapping `cache_clear`. I may be able to find a better name for the private inner function though.